android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -73,13 +80,58 @@ private void downloadFeed() {
             return;
```

Comment:
Here you are returning after only checking condition for Facebook. It will not fetch twitter feed if facebook link is not available